internal/install/execution/go_task_recipe_executor.go

```diff
@@ -233,26 +234,31 @@ func varFromPrompt(envConfig recipes.VariableConfig) (string, error) {
 		msg = envConfig.Prompt
 	}
 
-	templates := &promptui.PromptTemplates{
-		Prompt:  "{{ . | bold }} ",
-		Valid:   "{{ . | bold }} ",
-		Invalid: "{{ . | bold }} ",
-		Success: "  - {{ . }} ",
-	}
-
-	prompt := promptui.Prompt{
-		Label:     msg,
-		Templates: templates,
-	}
+	value := ""
+	var prompt survey.Prompt
 
 	if envConfig.Secret {
-		prompt.HideEntered = true
-		prompt.Mask = '*'
+		prompt = &survey.Password{
+			Message: msg,
+		}
```

Comment:
Does this automagically mask the input with `*****`?

Reply:
It does.  You can see an example here: https://github.com/AlecAivazis/survey/#password

Reply:
Feel free to check the branch out and give it a test though.

Reply:
Ah cool 🙌🏼